models/articles.model.js

```diff
@@ -1,6 +1,23 @@
 const db = require('../db/connection')
-exports.fetchArticles = () => {
-    return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id)::INTEGER AS comment_count FROM articles LEFT JOIN comments ON comments.article_id=articles.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;`)
+exports.fetchArticles = (query) => {
+    const queries = [
+        "author",
+        "topic",
+        "sort_by",
+        "order"
+    ]
```

Comment:
lines 3-8, I don't think you're using this yet, if you're following TDD you should only be writing code that's required. If you passed the project to someone else to complete the next ticket, they might write duplicate code or get confused why theres code for _their_ ticket already.

Reply:
On second thoughts, you should be using this! But at this point, "topic" is the only validColumn you can filter for. I would perhaps rename the greenlist to validColumns. 

You might be handling "tpic" with psql but you are still vulnerable to SQL injection.